android-auto-app/src/main/java/com/mapbox/maps/testapp/auto/testing/CarJavaInterfaceChecker.java

```diff
@@ -9,11 +9,13 @@
 import com.mapbox.maps.EdgeInsets;
 import com.mapbox.maps.MapInitOptions;
 import com.mapbox.maps.MapSurface;
+import com.mapbox.maps.MapboxExperimental;
 import com.mapbox.maps.ScreenCoordinate;
 import com.mapbox.maps.extension.androidauto.MapboxCarMap;
 import com.mapbox.maps.extension.androidauto.MapboxCarMapObserver;
 import com.mapbox.maps.extension.androidauto.MapboxCarMapSurface;
 
+@MapboxExperimental
```

Comment:
why do this in this PR?

Reply:
commented same here : https://github.com/mapbox/mapbox-maps-android/pull/1118#discussion_r807803076